docs/src/main/sphinx/security/cert-auth.rst

```diff
@@ -0,0 +1,108 @@
+==========================
+Certificate authentication
+==========================
+
+Trino can be configured to support certified clients seeking validation from a
+Trino server. This authentication method is only provided to support sites that
+have an absolute requirement for client authentication *and already have* client
```

Comment:
Is there guidance in the Google Dev TW guide about emphasizing sentences using italics? I'm not sure its necessary: https://developers.google.com/style/text-formatting

Reply:
It is very necessary to make sure our readers know they must already have client certs, or they're barking up the wrong tree. Now, I'm open to suggestions on how we go about designating that making-sure.

Reply:
If its a blocker then wouldn't a caution/notice of some form be better to draw emphasis? https://developers.google.com/style/notices

Reply:
OK, you've swayed me. I put an Important note in the JWT page to narrow the readership, and so there should be one here. Caution, Notice, and Warning are all too strong -- they won't lose data if they configure cert auth, only time and brain cells.

Reply:
In other places we use **strongly suggest** etc. With bold. I suggest we stick with the bold convention for any emphasis. 

That having been said, I would also feel better about this being a separate callout in a warning if it is indeed that dire.